Website/src/items/content/EdgeWebsite/Data/CMP Data/Testimonies/Striva.yml

```diff
@@ -33,7 +36,7 @@ Languages:
         default\Anonymous
     - ID: "8cdc337e-a112-42fb-bbb4-4143751e123f"
       Hint: __Revision
-      Value: "38b28323-85a7-4cde-ba20-3cefbe516fb2"
+      Value: "0dc74ba1-a40f-46c8-bd22-e03c62459d62"
     - ID: "a4eb8a28-879d-41d9-87c9-2ed2f03d0ae4"
       Hint: Logo
```

Comment:
The DemoEdge URL in this Logo field is not valid anymore. Can you please update it to a playsummit URL?